src/caselawclient/xquery/set_metadata_court.xqy

```diff
@@ -2,22 +2,26 @@ xquery version "1.0-ml";
 
 declare namespace akn = "http://docs.oasis-open.org/legaldocml/ns/akn/3.0";
 declare namespace uk = "https://caselaw.nationalarchives.gov.uk/akn";
+
+let $proprietary-node := document($uri)/akn:akomaNtoso/akn:*/akn:meta/akn:proprietary/uk:court
```

Comment:
importantly this now has `akn:*` rather than `akn:judgment` to allow for press summaries and eventually other docs types which have <doc> tag instead of <judgment> tag